deploy/helm/flightctl/templates/flightctl-worker-config.yaml

```
@@ -0,0 +1,19 @@
+{{ if .Values.flightctl.worker.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flightctl-config
```

```suggestion
  name: flightctl-workers-config
```

I know they live in separate namespaces (the other flightctl-config), but I would want to make sure they work even if we used the same namespace, also make it clear under a list that they have different purposes.

---

OK